plugins/record-hook/src/RecordHook.ts

```diff
@@ -121,12 +121,10 @@ export const BulkRecordHook = async (
       }
       endTimer('filter modified records', debug)
 
-      if (debug) {
-        logInfo(
-          '@flatfile/plugin-record-hook',
-          `${modifiedRecords.length} modified records`
-        )
-      }
+      logInfo(
```

Comment:
Just checking this was intentional, do we want logs outside of debug?

Reply:
Yes, this was intentional